### ISSUE TITLE
Bootstrap 4 or Safari bug?

### ISSUE THREAD

**Issue by johnunclesam** [twbs/bootstrap#22645]
What do you think about this strange behaviour?

http://stackoverflow.com/questions/44025290/flexbox-strange-behaviour-with-cssgrid

It works good with Chrome or Firefox, not with Safari (iPad/iPhone devices and Mac).

Safari 10.1, latest today.

**Comment by Johann-S** [comment]
They are some issues on Safari 10 with flex support see : http://caniuse.com/#search=flexbox

**Comment by mdo** [comment]
Likely not a Bootstrap bug given it's an issue with CSS grid, but please holler if that's not the case.

**Comment by Goran7777** [comment]
I think safari must close his perfomances to the chrome and firefox in the near future,only on Safari bootstrap has a weird behavior.

**Comment by sankarnair123** [comment]
was this ever fixed , i am having issues with grids only on safari

**Comment by tuku-dev** [comment]
I have followed this [link](https://github.com/twbs/bootstrap/issues/24219#issuecomment-370356871), it has written that make .row:before, .row:after inline-block the problem will solve. I have added it. When i checked on safari, I found that :before and :after are not showing in it. so how could it solve the problem? Don't know what shall i do. I might change all thing to Bootstrap old version. Don't know why people use safari if there are several fastest browser.
